src/app/api.ts

```diff
@@ -0,0 +1,23 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+import { IExecDataProtector } from '@iexec/dataprotector';
+import { connect } from 'wagmi/actions';
+import { InjectedConnector } from '@wagmi/connectors/injected';
+
+export const api = createApi({
+  baseQuery: fetchBaseQuery(),
+  endpoints: () => ({}),
+});
+
+export const getIExecDataProtectorAndRefresh =
+  async (): Promise<IExecDataProtector> => {
+    try {
+      const result = await connect({
+        connector: new InjectedConnector() as any,
```

Comment:
wagmi is configured for a bunch of providers but you only use `InjectedConnector` it seems buggy

Reply:
Commit: 7bf8b83d30c9ea974e744bb7483f3b85797f3e8c

Reply:
in api.ts : 
- 'IExecDataProtector' is defined but never used
- 'getAccount' is declared but its value is never read.ts(6133)

Reply:
done : Commit: 35d4b6b533bf3c97a5bab6e32f31df32c61bfc1f